mlir/include/mlir/InitRocMLIRCLOptions.h

```diff
@@ -25,6 +26,16 @@ inline void registerMLIRCLOptions() {
   mlir::registerPassManagerCLOptions();
 }
 
+inline void registerRocMLIROptions() {
```

Comment:
`registerRocMLIRLibClOptions()` or some other name that makes it look less like an upstream function